src/content/reference/platform/sites/start-from-worker.md

```diff
@@ -0,0 +1,53 @@
+
+# Start from worker
+
+Workers Sites require [Wrangler](https://github.com/cloudflare/wrangler) - make sure to be on the [latest version](/quickstart/#updating-the-cli) - and the Workers [Unlimited plan](https://workers.cloudflare.com/sites#plans).
+
+Make sure to be on the latest version.
+
+If you have a pre-existing Worker project, you can use Workers Sites to serve static assets to the Worker. To do so, follow these instructions:
+
+1. Create a directory in the root of your project (e.g. `workers-site`) and add configuration to your `wrangler.toml` to point to it. Also add the path to your Worker script (probably `index.js`).
+
+    ```
```

Comment:
Set syntax `sh` and use frontmatter to set:

```
---
filename: wrangler.toml
---
```

(And remove `# wrangler.toml` comment)